documentation/staging/content/userguide/base-images/custom-images.md

```diff
@@ -12,29 +12,26 @@ description: "Create custom images and apply patches."
   - [Create a custom base image](#create-a-custom-base-image)
   - [Create a custom image with your domain inside the image](#create-a-custom-image-with-your-domain-inside-the-image)
   - [Create a custom image with your model inside the image](#create-a-custom-image-with-your-model-inside-the-image)
-- [Apply patched images to a running domain](#apply-patched-images-to-a-running-domain)
-  - [Domain in PV](#domain-in-pv)
-  - [Model in Image with auxiliary images](#model-in-image-with-auxiliary-images)
-  - [Model in Image without auxiliary images](#model-in-image-without-auxiliary-images)
-  - [Domain in Image](#domain-in-image)
 
 ### Use the WebLogic Image Tool to create custom images
 
-To create custom images for your domain resource, you can use the [WebLogic Image Tool](https://oracle.github.io/weblogic-image-tool/).
-Download and install the WebLogic Image Tool (WIT) following the WIT [Setup](https://oracle.github.io/weblogic-image-tool/userguide/setup/) instructions.
+You can use the [WebLogic Image Tool](https://oracle.github.io/weblogic-image-tool/) (WIT) to build your own WebLogic Server or
+Fusion Middleware Infrastructure images (with the latest Oracle Linux images, Java updates, and WebLogic Server patches),
+apply one-off patches to existing OCR images, or overlay your own files and applications on top of an OCR image.  
+
+Download and install the WebLogic Image Tool (WIT) following the
+WIT [Setup](https://oracle.github.io/weblogic-image-tool/userguide/setup/) instructions.
 Also, refer to the WIT [Quick Start](https://oracle.github.io/weblogic-image-tool/quickstart/quickstart/) Guide.
 
 The WebLogic Image Tool `create`, `update`, or `rebase` commands supply
 three different ways to generate a custom WebLogic Server installation image
-from a base OS image (optionally, with WebLogic patches).
-
-In addition, the WIT `createAuxImage` command
+from a base OS image (optionally, with WebLogic patches). In addition, the WIT `createAuxImage` command
 supports creating auxiliary images which
 do _not_ contain a WebLogic Server installation,
 and instead, solely contain the [WebLogic Deploy Tool](https://oracle.github.io/weblogic-deploy-tooling/) binary, model, or archive files;
 this option is designed for the Model in Image [domain home source type]({{< relref "/userguide/managing-domains/choosing-a-model/_index.md" >}}).
 
-Finally, you can use the WIT `inspect` command to [inspect images]({{< relref "/userguide/base-images/_index.md#inspect-images" >}}).
+Finally, you can use the WIT `inspect` command to [inspect images]({{< relref "/userguide/base-images/ocr-images#inspect-images" >}}).
```

Comment:
In the `rebase` bullet below, please add a new sub bullet at the end:  For more information, see [Apply patched images to a running domain]({{< relref "/userguide/base-images/patch-images#apply-patched-images-to-a-running-domain" >}}).  (Or point directly to the dii section of patch-images.)

Reply:
Done.